helm/nessie/values.yaml

```diff
@@ -97,19 +97,23 @@ authorization:
     # allowViewingBranch: op=='VIEW_REFERENCE' && role.startsWith('test_user') && ref.startsWith('allowedBranch')
     # allowCommits: op=='COMMIT_CHANGE_AGAINST_REFERENCE' && role.startsWith('test_user') && ref.startsWith('allowedBranch')
 
-jaegerTracing:
-  # -- Specifies whether jaeger tracing for the nessie server should be enabled.
+tracing:
+  # -- Specifies whether tracing for the nessie server should be enabled.
   enabled: false
-  # -- The traces endpoint, in case the client should connect directly to the Collector, e.g. http://jaeger-collector:14268/api/traces
-  endpoint: ""
-  # -- The Jaeger service name.
-  serviceName: nessie
-  # -- Whether metrics are published if tracing is enabled.
-  publishMetrics: true
-  # -- The sampler type (const, probabilistic, ratelimiting or remote).
-  samplerType: ratelimiting
-  # -- The request sampling probability. 1=Sample all requests. Set samplerParam to somewhere between 0 and 1, e.g. 0.50, if you do not wish to sample all requests.
-  samplerParam: 1
+  # -- The collector endpoint URL to connect to (required).
+  # The endpoint URL must have either the http:// or the https:// scheme.
+  # The collector must talk the OpenTelemetry protocol (OTLP) and the port must be its gRPC port (by default 4317).
+  # See https://quarkus.io/guides/opentelemetry for more information.
+  endpoint: "http://otlp-collector:4317"
+  # -- Which requests should be sampled. Valid values are: "all", "none", or a ratio between 0.0 and
+  # 1.0 (inclusive). E.g. 0.5 means that 50% of the requests will be sampled.
+  sample: all
+  # -- Resource attributes to identify the nessie service among other tracing sources.
+  # See https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/#service.
+  # If left empty, the following attribute will be automatically added: service.name=nessie.
```

Comment:
I am having second thoughts about this.

If `service.name` is not provided at all, it seems that Quarkus will use the value from `quarkus.application.name`, which in our case is `Nessie`. This might be enough, wdyt?

Reply:
Defaulting to `quarkus.application.name` SGTM.

Reply:
OK I made the change. You will need to re-approve, sorry.